src/App/pages/components/Header/Header.js

```diff
@@ -0,0 +1,22 @@
+import film from './film.png';
```

Comment:
Create a separate folder for images at the root of your project ('src' folder) and store all the images there.

Reply:
fixed